src/Template/Bake/Template/index.ctp

```diff
@@ -39,20 +39,21 @@ $fields = collection($fields)
     endforeach;
 %>
     </ul>
-</div>
-<div class="<%= $pluralVar %> index large-10 medium-9 columns">
+</nav>
+<div class="<%= $pluralVar %> index large-9 medium-8 columns" id="content">
```

Comment:
@mtkocak The benefit of basing the css off a standard css lib (zurb foundation in this case) is that people can easily add new elements as per their app needs.